scripts/pipeline/populate_per_genome_database.pl

```diff
@@ -0,0 +1,291 @@
+
+    my $genome_dba = $to_dba->get_GenomeDBAdaptor;
+    $genome_dba->store( $genome_db );
+}
+
+=head2 copy_mlss_and_ss
+
+    Arg[1]      : Bio::EnsEMBL::Compara::DBSQL::DBAdaptor $from_dba (Mandatory)
+    Arg[2]      : Bio::EnsEMBL::Compara::DBSQL::DBAdaptor $to_dba (Mandatory)
+    Arg[3]      : Bio::EnsEMBL::Compara::DBSQL::GenomeDBAdaptor $genome_db (Mandatory)
+    Description : copy from $from_dba to $to_dba just the method_link_species_set
+                  and species_sets which correspond to $genome_db_id only.
+    Returns     : None
+    Exceptions  : None
+
+=cut
+
+sub copy_mlss_and_ss {
+    my ($from_dba, $to_dba, $genome_db) = @_;
+
+    my $from_mlss_adap = $from_dba->get_MethodLinkSpeciesSetAdaptor;
+    my $to_mlss_adap   = $to_dba->get_MethodLinkSpeciesSetAdaptor;
+
+    my $mlsss = $from_mlss_adap->fetch_all_by_GenomeDB( $genome_db );
+    foreach my $mlss ( @$mlsss ) {
+        $to_mlss_adap->store( $mlss );
+    }
+}
+
+=head2 copy_gene_and_seq_members
+
+    Arg[1]      : Bio::EnsEMBL::Compara::DBSQL::DBAdaptor $from_dba (Mandatory)
+    Arg[2]      : Bio::EnsEMBL::Compara::DBSQL::DBAdaptor $to_dba (Mandatory)
+    Arg[3]      : Bio::EnsEMBL::Compara::DBSQL::GenomeDBAdaptor $genome_db (Mandatory)
+    Arg[4]      : $dnafrag default: undef (Optional)
+    Description : copy from $from_dba to $to_dba just the gene_member and
+                  seq_members which correspond to $genome_db_id only. Optionally
+                  copies across dnafrags if required.
+    Returns     : None
+    Exceptions  : None
+
+=cut
+
+sub copy_gene_and_seq_members {
+    my ($from_dba, $to_dba, $genome_db, $dnafrag) = @_;
+
+    my $from_gene_adap = $from_dba->get_GeneMemberAdaptor;
+    my $to_gene_adap   = $to_dba->get_GeneMemberAdaptor;
+
+    my $from_dna_adap = $from_dba->get_DnaFragAdaptor;
+    my $to_dna_adap   = $to_dba->get_DnaFragAdaptor;
+
+    my $from_sm_adap = $from_dba->get_SeqMemberAdaptor;
+    my $to_sm_adap   = $to_dba->get_SeqMemberAdaptor;
+
+    my $from_seq_adap = $from_dba->get_SequenceAdaptor;
+    my $to_seq_adap   = $to_dba->get_SequenceAdaptor;
+
+    my $seq_members = $from_sm_adap->fetch_all_canonical_by_GenomeDB( $genome_db );
+    foreach my $seq_member ( @$seq_members ) {
+        # For some reason $seq_member->gene_member returns undef attribute.
+        # May be linked to intentional missing dnafrag for homology etc.
+        my $gene_member = $from_gene_adap->fetch_by_dbID( $seq_member->gene_member_id );
+        $to_gene_adap->store( $gene_member );
+        my $sequence = $from_seq_adap->fetch_by_dbID( $seq_member->sequence_id );
+        $to_seq_adap->store( $sequence );
+        $to_sm_adap->store( $seq_member );
+        # For rapid release as of e103-e104 only canonical peptides are used -
+        # when pairwise alignments are introduced, dnafrags will also need to be copied
+        if ( $dnafrag ) {
+            copy_dnafrags( $from_dba, $to_dba, $genome_db->dbID );
+        }
```

Comment:
I think this should be moved to the main script body.
```suggestion
        # For rapid release as of e103-e104 only canonical peptides are used -
        # when pairwise alignments are introduced, dnafrags will also need to be copied
        if ( $dnafrag ) {
            copy_dnafrags( $from_dba, $to_dba, $genome_db );
        }
```